examples/server-demo/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@algolia/recommend-server-example",
+  "description": "Recommend Server example",
+  "version": "1.7.1",
+  "private": true,
+  "license": "MIT",
+  "scripts": {
+    "start-server": "webpack && nodemon ./dist/server.js"
+  },
+  "dependencies": {
+    "@algolia/client-search": "4.10.5",
+    "@algolia/recommend": "4.13.0",
+    "@algolia/recommend-react": "1.8.0",
+    "algoliasearch": "4.10.5",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
+    "express": "4.17.1"
+  },
+  "devDependencies": {
+    "@babel/core": "7.4.5",
```

Comment:
if we use the same versions as used in the root already it avoids duplicating in yarn.lock

Reply:
we don't have @babel/core in the root

Reply:
I just see that it added some things to the lock file that could be harmonised / made compatible if this was ^7.4.5 or the exact version used somewhere else in the project. I guess it's not a big problem though